WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -322,11 +322,11 @@ private string PreComposeText(string input)
 				{
 					var start = Math.Min(selectionStart, selectionEnd);
 					var end = Math.Max(selectionStart, selectionEnd);
-					preComposedText = preComposedText.Substring(0, start) + preComposedText.Substring(end);
+					preComposedText = preComposedText.Substring(0, start) + preComposedText[end..];
```

Comment:
Why only to the end part?

Reply:
VS didn't complain about that and suggested to use the Range operator only on the end part.

Reply:
If we are touching this, use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) instead of concatenation.

Reply:
@molnard Is something like this better?
`preComposedText = $"{preComposedText.Substring(0, start)}{preComposedText[end..]}";`

Reply:
I don't know what happened with my previous feedback so, there go go again:

We don't fix only what VS complains about because we are the carpenters and VS is just the hammer. Make the change consistent. It is okay to use `Substring` and it is also okay to  use range operators but it is not okay to mix them and even worse to mix them without understanding why (or because VS says so).

Reply:
I pushed changes.